doc/tutorials/planning_around_objects/planning_around_objects.rst

```diff
@@ -0,0 +1,123 @@
+Planning Around Objects
+=======================
+
+This tutorial will introduce you to inserting objects into the planning scene and planning around them.
+
+Prerequisites
+-------------
+
+If you haven't already done so, make sure you've completed the steps in :doc:`Visualizing in RViz </doc/tutorials/visualizing_in_rviz/visualizing_in_rviz>`.
+This project assumes you are starting with the ``hello_moveit`` project, where the previous tutorial left off.
+
+Steps
+-----
+
+1 Add include for Planning Scene Interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At the top of your source file add this to the list of includes:
+
+.. code-block:: C++
+
+  #include <moveit/planning_scene_interface/planning_scene_interface.h>
+
+2 Change the Target Pose
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+First update the target pose with this one change to make the robot plan to a different location.
+
+.. code-block:: C++
+
+    // Set a target Pose
+    auto const target_pose = [] {
+      geometry_msgs::msg::Pose msg;
+      msg.orientation.w = 1.0;
+      msg.position.x = 0.28;
+      msg.position.y = 0.4;  // <---- This value was changed
+      msg.position.z = 0.5;
+      return msg;
+    }();
+    move_group_interface.setPoseTarget(target_pose);
+
+3 Create a Collision Object
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the next block of code we create a collision object.
+The first thing to notice is that it is being place in the frame of the robot.
+If we had a preception system that reported the location of objects in our scene that we wanted to plan around this is the sort of message it would build.
+As this is just an example we are creating it manually.
```

Comment:
This one is so nit picky and I'm sorry 😆 https://dictionary.cambridge.org/us/grammar/british-grammar/as-because-or-since
```suggestion
Because this is just an example, we are creating it manually.
```